src/components/StatePicker.js

```diff
@@ -17,11 +17,15 @@ const propTypes = {
     /** The value that needs to be selected */
     value: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
 
+    /** Should the input be styled for errors  */
```

Comment:
NAB: I think this comment could be slightly improved. We are not styling for errors in _this_ component, so I think it should be changed to something less specific.

Reply:
Or we could reference the `propTypes` of the `ExpensiPicker`? It looks like this is just a wrapper around this component.

Reply:
I guess this applies to all props in `StatePicker`, they are all passed down to `ExpensiPicker`.

Do you have any suggestion/example for a "less specific" comment or a comment referencing a `propType` of another component?

I tried to search for examples of referencing another propType while documenting in our codebase and in google and didn't find any. Would a reference to the `propTypes` of the `ExpensiPicker` look like: `See ExpensiPicker "hasError" prop documentation`?

Reply:
To clarify, I'm suggesting that we simply import the propTypes for use here.

Reply:
I see, I just noticed that we do something like that for `BaseExpensiTextInput` and `ExpensiTextInput`. Both import the `propTypes` from the same file, and `ExpensiTextInput` is a wrapper for `BaseExpensiTextInput`.